tutorials/io/mergeSelective.py

```diff
@@ -0,0 +1,118 @@
+# \file
+# \ingroup tutorial_io
+# \notebook -nodraw
+# Merge only part of the content of a set of files.
+# This macro demonstrates how to merge only a part of the content of a set
+# of input files, specified via the interface.
+# ~~~{.cpp}
+#     TFileMerger::AddObjectNames(const char *names)
+# ~~~
+# The method can be called several times to add object names, or using a single
+# string with names separated by a blank. Directory names contained in the files
+# to be merged are accepted.
+#
+# Two modes are supported:
+# 1. kOnlyListed: via <tt>TFileMerger::PartialMerge(kOnlyListed)</tt>
+#    This will merge only the objects in the files having the names in the
+#    specified list. If a folder is specified, its whole content will be merged
+#
+# 2. kSkipListed: via <tt>TFileMerger::PartialMerge(kSkipListed)</tt>
+#    This will skip merging of specified objects. If a folder is specified, its
+#    whole content will be skipped. 
```

Comment:
This explanation is unclear to me. How are `kOnlyListed` and `kSkipListed` options passed by the user? The mentioned function `AddObjectNames` accepts a `const char *` argument. I understand this was just copy-pasted, let's just try to give the minimum information required to understand the tutorial